services/payments/src/simcore_service_payments/services/stripe.py

```diff
@@ -0,0 +1,66 @@
+""" Interface to communicate with the Resource Usage Tracker (RUT)
+
+- httpx client with base_url to PAYMENTS_RESOURCE_USAGE_TRACKER
+
+"""
+
+import logging
+
+import httpx
+from fastapi import FastAPI
+from models_library.payments import StripeInvoiceID
+from servicelib.fastapi.app_state import SingletonInAppStateMixin
+from servicelib.fastapi.http_client import (
+    AttachLifespanMixin,
+    BaseHTTPApi,
+    HealthMixinMixin,
+)
+
+from ..core.settings import ApplicationSettings
+from ..models.stripe import InvoiceData
+
+_logger = logging.getLogger(__name__)
+
+
+class _StripeBearerAuth(httpx.Auth):
+    def __init__(self, token):
+        self._token = token
+
+    def auth_flow(self, request):
+        request.headers["Authorization"] = f"Bearer {self._token}"
+        yield request
+
+
+class StripeApi(
+    BaseHTTPApi, AttachLifespanMixin, HealthMixinMixin, SingletonInAppStateMixin
+):
+    app_state_name: str = "stripe_api"
+
+    async def http_check_connection(
+        self,
+    ) -> bool:
+        response = await self.client.get("/v1/products")
+        response.raise_for_status()
+
+        return True
```

Comment:
Here the logic is mixed since if success **always** receives true while failure raises ... 
Why don't you override or extend the `HealthMixinMixin` instead. I think you are just inheriting but not using it.
IMO what you intend here is a healthy checkpoint.  See `HealthMixinMixin` and read also the links provided about Liveness and Readiness probes in packages/models-library/src/models_library/healthchecks.py